portable/MemMang/heap_4.c

```diff
@@ -288,7 +288,7 @@ void * pvPortMalloc( size_t xWantedSize )
                     }
                     else
                     {
-                        mtCOVERAGE_TEST_MARKER();
+                        xWantedSize = pxBlock->xBlockSize;
```

Comment:
Instead of reusing `xWantedSize`, can we please create a new variable `xAllocatedBlockSize`? This does not change the logic but helps in readability.

Reply:
Good idea.